openapi/openapi.yaml

```
@@ -508,6 +508,42 @@ components:
         - id
         - title
         - abbr
+    CapitalCommitment:
+      type: object
+      properties:
+        id:
+          type: string
+          description: A uuid used to refer to the capital commitment. 
+          format: uuid
+        type:
+          type: string
+          description: A four character string used to refer to the commitment type.
+          pattern: '^([A-z]{4})$'
+          example: DSGN
+        plannedDate:
+          type: date
+          description: A string used to refer to the date when the commitment is projected to be committed.
+          example: '2012-04-23T18:25:43.511Z'
+        budgetLineCode:
+          type: string
+          description: A string used to refer to the budget line.
+          example: "HW"
+        budgetLineId: 
+          type: string
+          description: A string used to refer to the budget line.
+          example: '0002Q'
+        sponsoringAgencyInitials:
+          type: string
+          description: A string of variable length containing the initials of the sponsoring agency.
+          example: DOT
+        budgetType:
+          type: string
+          description: A string of variable length denoting the type of budget.
+          example: 'Highways'
+        totalValue:
+          type: number
+          description: A numeric string used to refer to the amount of total planned commitments.
+          example: 1600000
```

Need to add "required" addendum

```suggestion
          example: 1600000
          required:
            - id
            - type
            - plannedDate
            - budgetLineCode
            - budgetLineId
            - sponsoringAgencyInitials
            - budgetType
            - totalValue
```